conan/api/subapi/graph.py

```diff
@@ -87,7 +89,7 @@ def load_root_test_conanfile(self, path, tested_reference, profile_host, profile
         return root_node
 
     @api_method
-    def load_root_virtual_conanfile(self, profile_host, requires=None, tool_requires=None):
+    def _load_root_virtual_conanfile(self, profile_host, requires=None, tool_requires=None):
```

Comment:
Should this be marked as an API method and have an underscore? I thought underscore meant private function? 🤔

Reply:
Probably not, the "virtual" conanfile seems a bit of implementation detail, I don't see a clear use case of users having to do that, now that we exposed a ``load_graph_consumer(path, ...)`` and ``load_graph_requires(requires, tool_requires...)`` methods. We don't need to do it now, lets keep the API minimal to the construction of our own built-in commands, and open it as necessary when requested by users.

Reply:
I agree it should be private, thats why I am confused why it's `@api_method` :)

Reply:
Oh, I see, you are right, that ``api_method`` should be removed.